curiouslearning/FeedTheMonsterJS
# Fix CR-67: Development build Tigragna

# Changes
- tigragna language folder

# How to test
- Check Tigragna new added language and feedback text/audios.

Ref: [CR-67](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-67)


[CR-67]: https://curiouslearning.atlassian.net/browse/CR-67?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
